scripts/upgrade_scripts/upgrade-fluffy.sh

```diff
@@ -4,17 +4,26 @@
 #   upgrade script from flufflys timer   #
 ##########################################
 
-# change this setting to your install directory
-SERVERDIR="/srv/csgo-server/csgo"
+# change this to your csgo directory, if not running from "upgrade_scripts" folder
+SERVERDIR="../.."
 
 rm -rf "${SERVERDIR}/sound/quake"
-rm "${SERVERDIR}/sound/surftimer/*.mp3"
+rm -f "${SERVERDIR}/sound/surftimer/*.mp3"
```

Comment:
do we really need to force it?, same applies to the other lines in this file in general